docs/source/user/settings.rst

```diff
@@ -180,6 +180,41 @@ If this option and organization change page option are both
 empty, then all freeradius API requests for the organization
 will return ``403``.
 
+.. _coa_enabled_setting:
+
+``OPENWISP_RADIUS_COA_ENABLED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``False```
+
+If set to ``True``, openwisp-radius will update the NAS with the
+user's current RADIUS attributes whenever the ``RadiusGroup`` of
+user is changed. This allow enforcing of rate limits on active
+RADIUS sessions without requiring users to re-authenticate. For
+more details, :ref:`read the dedicated section for configuring
+openwisp-radius and NAS for using CoA <change_of_authorization>`.
+
+This can be overridden for each organization separately
+via the organization radius settings section of the admin interface.
+
+.. image:: /images/organization_coa_enabled.png
+   :alt: CoA enabled
+
+```RADCLIENT_ATTRIBUTE_DICTIONARIES```
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+----------+
+| **type**:    | ``list`` |
++--------------+----------+
+| **default**: |  ``[]``  |
++--------------+----------+
+
+List of absolute file path of RADIUS dictionaries used
+for RADIUS attribute mapping.
+
+You can refer this `example dictionary <https://github.com/openwisp/openwisp-radius/blob/master/tests/openwisp2/radclient_dictionary.txt>`_
+to get started.
```

Comment:
Let's ammend here as discussed (have the default dictionary automatically included here and allow other dictionaries to be included).